conf/aws.conf

```diff
@@ -0,0 +1,381 @@
+    #
+
+    keyName: %(keyName)s
+
+    #
+    # ID of the VPC subnet
+    # See: http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Subnets.html
+    #
+
+    subnetId: %(subnetId)s
+
+    #
+    # Comma separated list of security group IDs
+    # See: http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_SecurityGroups.html
+    #
+
+    securityGroupsIds: %(securityGroupsIds)s
+
+    #
+    # A prefix that Cloudera Director should use when naming the instances (this is not part of the hostname)
+    #
+
+    instanceNamePrefix: cloudera-director
+
+    #
+    # Specify a size for the root volume (in GBs). Cloudera Director will automatically expand the
+    # filesystem so that you can use all the available disk space for your application
+    # See: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/storage_expand_partition.html
+    #
+
+    rootVolumeSizeGB: 100 # defaults to 50 GB if not specified
+
+    #
+    # Specify the type of the EBS volume used for the root partition. Defaults to gp2
+    # See: http://aws.amazon.com/ebs/details/
+    #
+
+    # rootVolumeType: gp2 # OR standard (for EBS magnetic)
+
+    #
+    # Whether to associate a public IP address with instances or not. If this is false
+    # we expect instances to be able to access the internet using a NAT instance
+    #
+    # Currently the only way to get optimal S3 data transfer performance is to assign
+    # public IP addresses to your instances and not use NAT (public subnet type of setup)
+    #
+    # See: http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/vpc-ip-addressing.html
+    #
+
+    associatePublicIpAddresses: true
+
+}
+
+#
+# SSH credentials to use to connect to the instances
+#
+
+ssh {
+    username: ec2-user # for RHEL image
+    privateKey: /home/ec2-user/id.pem # with an absolute path to .pem file
+}
+
+#
+# A list of instance types to use for group of nodes or management services
+#
+
+instances {
+    m1x {
+        type: m1.xlarge
+
+        #
+        # Amazon Machine Image (AMI)
+        #
+        # See: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html
+        # Compatibility matrix: https://aws.amazon.com/amazon-linux-ami/instance-type-matrix/
+        #
+        # Red Hat Enterprise Linux AMI IDs: http://aws.amazon.com/partners/redhat/
+        # We support Red Hat Enterprise Linux 6.4 (64bit) 64bit PV or HVM
+        #
+
+        image: %(image)s
+
+        #
+        # Name of the IAM Role to use for this instance type
+        # http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html
+        #
+
+        # iamProfileName: REPLACE-ME
+
+        tags {
+            owner: %(keyName)s
+        }
+
+        bootstrapScript: """#!/bin/sh
+
+# This is an embedded bootstrap script that runs as root and can be used to customize
+# the instances immediately after boot and before any other Cloudera Director action
+
+# If the exit code is not zero Cloudera Director will automatically retry
+
```

Comment:
Should we put some of our eggo-specific init here?  Or continue to leave it to fabric?  Probably the latter...